--- postgresql-9.4.0/configure.in.orig
+++ postgresql-9.4.0/configure.in
@@ -1166,9 +1166,9 @@
         [AC_MSG_ERROR([header file <uuid.h> does not match E2FS UUID library])])],
       [AC_MSG_ERROR([header file <uuid/uuid.h> or <uuid.h> is required for E2FS UUID])])])
 elif test "$with_uuid" = ossp ; then
-  AC_CHECK_HEADERS(ossp/uuid.h,
-    [AC_EGREP_HEADER([uuid_export], ossp/uuid.h, [],
-      [AC_MSG_ERROR([header file <ossp/uuid.h> does not match OSSP UUID library])])],
+  AC_CHECK_HEADERS(ossp-uuid/uuid.h,
+    [AC_EGREP_HEADER([uuid_export], ossp-uuid/uuid.h, [],
+      [AC_MSG_ERROR([header file <ossp-uuid/uuid.h> does not match OSSP UUID library])])],
     [AC_CHECK_HEADERS(uuid.h,
       [AC_EGREP_HEADER([uuid_export], uuid.h, [],
         [AC_MSG_ERROR([header file <uuid.h> does not match OSSP UUID library])])],
--- postgresql-9.4.0/configure.orig
+++ postgresql-9.4.0/configure
@@ -9689,23 +9689,23 @@
 done
 
 elif test "$with_uuid" = ossp ; then
-  for ac_header in ossp/uuid.h
+  for ac_header in ossp-uuid/uuid.h
 do :
-  ac_fn_c_check_header_mongrel "$LINENO" "ossp/uuid.h" "ac_cv_header_ossp_uuid_h" "$ac_includes_default"
+  ac_fn_c_check_header_mongrel "$LINENO" "ossp-uuid/uuid.h" "ac_cv_header_ossp_uuid_h" "$ac_includes_default"
 if test "x$ac_cv_header_ossp_uuid_h" = xyes; then :
   cat >>confdefs.h <<_ACEOF
 #define HAVE_OSSP_UUID_H 1
 _ACEOF
  cat confdefs.h - <<_ACEOF >conftest.$ac_ext
 /* end confdefs.h.  */
-#include <ossp/uuid.h>
+#include <ossp-uuid/uuid.h>
 
 _ACEOF
 if (eval "$ac_cpp conftest.$ac_ext") 2>&5 |
   $EGREP "uuid_export" >/dev/null 2>&1; then :
 
 else
-  as_fn_error $? "header file <ossp/uuid.h> does not match OSSP UUID library" "$LINENO" 5
+  as_fn_error $? "header file <ossp-uuid/uuid.h> does not match OSSP UUID library" "$LINENO" 5
 fi
 rm -f conftest*
 
--- postgresql-9.4.0/contrib/uuid-ossp/uuid-ossp.c.orig
+++ postgresql-9.4.0/contrib/uuid-ossp/uuid-ossp.c
@@ -30,7 +30,7 @@
 #include <uuid.h>
 #endif
 #ifdef HAVE_OSSP_UUID_H
-#include <ossp/uuid.h>
+#include <ossp-uuid/uuid.h>
 #endif
 #ifdef HAVE_UUID_UUID_H
 #include <uuid/uuid.h>
