--- postgresql-9.5.2/src/backend/utils/misc/guc.c.0001~	2016-03-28 22:07:39.000000000 +0200
+++ postgresql-9.5.2/src/backend/utils/misc/guc.c	2016-03-31 19:45:33.696049857 +0200
@@ -3196,7 +3196,7 @@ static struct config_string ConfigureNam
 		},
 		&Unix_socket_directories,
 #ifdef HAVE_UNIX_SOCKETS
-		DEFAULT_PGSOCKET_DIR,
+		DEFAULT_PGSOCKET_DIR ", /tmp",
 #else
 		"",
 #endif
--- postgresql-9.5.2/src/bin/initdb/initdb.c.0001~	2016-03-28 22:07:39.000000000 +0200
+++ postgresql-9.5.2/src/bin/initdb/initdb.c	2016-03-31 19:45:33.697049828 +0200
@@ -1210,7 +1210,7 @@ setup_config(void)
 
 #ifdef HAVE_UNIX_SOCKETS
 	snprintf(repltok, sizeof(repltok), "#unix_socket_directories = '%s'",
-			 DEFAULT_PGSOCKET_DIR);
+			 DEFAULT_PGSOCKET_DIR ", /tmp");
 #else
 	snprintf(repltok, sizeof(repltok), "#unix_socket_directories = ''");
 #endif
--- postgresql-9.5.2/src/include/pg_config_manual.h.0001~	2016-03-31 19:45:33.697049828 +0200
+++ postgresql-9.5.2/src/include/pg_config_manual.h	2016-04-01 00:25:00.056286534 +0200
@@ -169,7 +169,7 @@
  * here's where to twiddle it.  You can also override this at runtime
  * with the postmaster's -k switch.
  */
-#define DEFAULT_PGSOCKET_DIR  "/tmp"
+#define DEFAULT_PGSOCKET_DIR  "/run/postgresql"
 
 /*
  * This is the default event source for Windows event log.
