diff -up postgresql-18.6/src/backend/utils/misc/postgresql.conf.sample.omv~ postgresql-18.6/src/backend/utils/misc/postgresql.conf.sample
--- postgresql-18.6/src/backend/utils/misc/postgresql.conf.sample.omv~	2026-08-14 00:00:00.000000000 +0000
+++ postgresql-18.6/src/backend/utils/misc/postgresql.conf.sample	2026-08-14 00:00:00.000000000 +0000
@@ -249,7 +249,7 @@
 #full_page_writes = on			# recover from partial page writes
 #wal_log_hints = off			# also do full page writes of non-critical updates
 					# (change requires restart)
-#wal_compression = off			# enables compression of full-page writes;
+wal_compression = zstd			# enables compression of full-page writes;
 					# off, pglz (or "on"), lz4, or zstd
 #wal_init_zero = on			# zero-fill new WAL files
 #wal_recycle = on			# recycle WAL files
