--- postgresql-9.2.3/configure.0000~	2013-02-04 22:28:13.000000000 +0100
+++ postgresql-9.2.3/configure	2013-02-11 17:57:31.589871470 +0100
@@ -13973,7 +13973,7 @@ fi
 # for contrib/uuid-ossp
 if test "$with_ossp_uuid" = yes ; then
 
-for ac_header in ossp/uuid.h
+for ac_header in ossp-uuid/uuid.h
 do
 as_ac_Header=`$as_echo "ac_cv_header_$ac_header" | $as_tr_sh`
 if { as_var=$as_ac_Header; eval "test \"\${$as_var+set}\" = set"; }; then
@@ -14269,8 +14269,8 @@ as_val=`eval 'as_val=${'$as_ac_Header'}
 _ACEOF
 
 else
-  { { $as_echo "$as_me:$LINENO: error: header file <ossp/uuid.h> or <uuid.h> is required for OSSP-UUID" >&5
-$as_echo "$as_me: error: header file <ossp/uuid.h> or <uuid.h> is required for OSSP-UUID" >&2;}
+  { { $as_echo "$as_me:$LINENO: error: header file <ossp-uuid/uuid.h> or <uuid.h> is required for OSSP-UUID" >&5
+$as_echo "$as_me: error: header file <ossp-uuid/uuid.h> or <uuid.h> is required for OSSP-UUID" >&2;}
    { (exit 1); exit 1; }; }
 fi
 
--- postgresql-9.2.3/configure.in.0000~	2013-02-11 17:57:31.589871470 +0100
+++ postgresql-9.2.3/configure.in	2013-02-11 17:58:12.416385183 +0100
@@ -1079,9 +1079,9 @@ fi
 
 # for contrib/uuid-ossp
 if test "$with_ossp_uuid" = yes ; then
-  AC_CHECK_HEADERS(ossp/uuid.h, [], [
+  AC_CHECK_HEADERS(ossp-uuid/uuid.h, [], [
     AC_CHECK_HEADERS(uuid.h, [],
-      [AC_MSG_ERROR([header file <ossp/uuid.h> or <uuid.h> is required for OSSP-UUID])])])
+      [AC_MSG_ERROR([header file <ossp-uuid/uuid.h> or <uuid.h> is required for OSSP-UUID])])])
 fi
 
 if test "$PORTNAME" = "win32" ; then
--- postgresql-9.2.3/contrib/uuid-ossp/uuid-ossp.c.0000~	2013-02-04 22:28:13.000000000 +0100
+++ postgresql-9.2.3/contrib/uuid-ossp/uuid-ossp.c	2013-02-11 17:57:31.589871470 +0100
@@ -21,7 +21,7 @@
  * what other systems do.
  */
 #ifdef HAVE_OSSP_UUID_H
-#include <ossp/uuid.h>
+#include <ossp-uuid/uuid.h>
 #else
 #ifdef HAVE_UUID_H
 #include <uuid.h>
