diff -up postgresql-15.4/src/backend/utils/adt/xml.c.omv~ postgresql-15.4/src/backend/utils/adt/xml.c
--- postgresql-15.4/src/backend/utils/adt/xml.c.omv~	2024-04-09 00:26:51.798793075 +0200
+++ postgresql-15.4/src/backend/utils/adt/xml.c	2024-04-09 00:29:41.828600849 +0200
@@ -119,7 +119,7 @@ struct PgXmlErrorContext
 
 static xmlParserInputPtr xmlPgEntityLoader(const char *URL, const char *ID,
 										   xmlParserCtxtPtr ctxt);
-static void xml_errorHandler(void *data, xmlErrorPtr error);
+static void xml_errorHandler(void *data, const struct _xmlError *error);
 static void xml_ereport_by_code(int level, int sqlcode,
 								const char *msg, int errcode);
 static void chopStringInfoNewlines(StringInfo str);
@@ -1749,7 +1749,7 @@ xml_ereport(PgXmlErrorContext *errcxt, i
  * Error handler for libxml errors and warnings
  */
 static void
-xml_errorHandler(void *data, xmlErrorPtr error)
+xml_errorHandler(void *data, const struct _xmlError *error)
 {
 	PgXmlErrorContext *xmlerrcxt = (PgXmlErrorContext *) data;
 	xmlParserCtxtPtr ctxt = (xmlParserCtxtPtr) error->ctxt;
